website/docs/proxy/index.md

```diff
@@ -6,12 +6,365 @@ tags: [podman-desktop, installing, windows, macos, linux]
 keywords: [podman desktop, containers, podman, installing, installation, windows, macos, linux]
 ---
 
-# Using Podman Desktop behind a proxy
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-You can use Podman Desktop behind a proxy in various situations, such as:
+# Installing and using Podman Desktop and Podman in a restricted environment
 
-- [Using a proxy on Windows and macOS](/docs/proxy/using-a-proxy)
-- [Using a proxy requiring a custom Certificate Authorities on macOS and Windows](/docs/proxy/using-a-proxy-requiring-a-custom-ca)
-- [Using a proxy on Linux](/docs/proxy/using-a-proxy-on-linux)
-- [Using a proxy in your containers](/docs/proxy/using-a-proxy-in-your-containers)
-- [Accessing resources behind a VPN with Podman on Windows](/docs/proxy/using-a-vpn-on-windows)
+In a restricted environment you might face this challenges:
+
+- The default Podman Desktop and Podman installation methods download assets during the setup.
+
+  However, a network restricted environment might refuse access to these external resources.
+
+  Consider rather using a specific installation method.
+
+- The Podman Machine receives a network address distinct to your computer network address.
+
+  However, resources in your VPN might be available only for the traffic going through your computer network address.
+
+  Consider enabling the **User mode networking** option when creating your Podman Machine to route the network traffic through your VPN.
+
+- The Podman Machine connects directly to the external network.
+
+  However, a restricted environment might block all traffic no going to a proxy.
+
+  Consider configuring Podman Desktop and your Podman Machine to route the traffic through a proxy.
+
+This tutorial is guiding you through the required steps to work in a restricted environment.
+
+## Installing Podman Desktop and Podman
+
+<Tabs groupId="operating-systems">
+<TabItem value="win" label="Windows">
+
+On Windows, to install Podman Desktop and Podman in a restricted environment, consider using the installer for restricted environments.
+
+- It has all artifacts required to install Podman Desktop and Podman.
+- It does not require Internet access to download resources during installation.
+- However, it does not contain additional utilities, such as Compose or Kind.
+
+1. Check that your environment has:
+
+   - 6 GB RAM for the Podman Machine.
+   - Windows Subsystem for Linux version 2 (WSL 2) prerequisites. See [Enabling WSL 2](https://docs.microsoft.com/en-us/windows/wsl/install), [WSL basic commands](https://learn.microsoft.com/en-us/windows/wsl/basic-commands), and [Troubleshooting WSL 2](https://learn.microsoft.com/en-us/windows/wsl/troubleshooting#error-0x80370102-the-virtual-machine-could-not-be-started-because-a-required-feature-is-not-installed):
+     - The Windows user has administrator privileges.
```

Comment:
Implied by previous step (you require admin for wsl install)

Reply:
Yes, that's a check *before* trying to install WSL.